sklearn/pipeline.py

```diff
@@ -481,7 +481,7 @@ def _inverse_transform(self, X):
         return Xt
 
     @if_delegate_has_method(delegate='_final_estimator')
-    def score(self, X, y=None):
+    def score(self, X, y=None, **score_params):
```

Comment:
Or we could have the signature as `..., sample_weight=None`, pass `**score_params` to `score` method and have it documented that it is applicable only if the underlying estimator supports it...